guides/micronaut-oracle-function-http/micronaut-oracle-function-http.adoc

```diff
@@ -353,6 +368,143 @@ and the output should look something like this (it should be the same as before
 }
 ----
 
+== Deploying as a Native Image
+
+=== Install GraalVM
+
+include::{commondir}/common-install-graal-function.adoc[]
+
+=== Building and deploying the native image
+
+Deploying the function as a native image is similar to the earlier deployment above.
+
+First you need to update your build script with the location to deploy the native image Docker container.
+
+:exclude-for-build:maven
+
+Edit `build.gradle` like before, but set the `images` property in the `dockerBuildNative` block this time, replacing `REGION`, `TENANCY`, and `REPO` as before:
+
+[source, groovy]
+.build.gradle
+----
+dockerBuildNative {
+    images = ["[REGION].ocir.io/[TENANCY]/[REPO]/$project.name-native:$project.version"]
+}
+----
+
+Since it's unlikely that you'll be deploying both jar-based containers and native image-based containers, you can use the same repo:
+
+[source, groovy]
+.build.gradle
+----
+dockerBuildNative {
+    images = ["[REGION].ocir.io/[TENANCY]/[REPO]/$project.name:$project.version"]
+}
+----
+
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+Since it's unlikely that you'll be deploying both jar-based containers and native image-based containers, you can use the same repo. If you wish to separate the native image containers, edit `pom.xml` and update the `jib.docker.image` property, appending "-native" to the value:
+
+[source, xml]
+.pom.xml
+----
+<jib.docker.image>[REGION].ocir.io/[TENANCY]/[REPO]/${project.artifactId}-native</jib.docker.image>
+----
+
+:exclude-for-build:
+
+Next, update the version.
+
+:exclude-for-build:maven
+
+Edit `build.gradle` and increment the version to `0.2`:
+
+[source, groovy]
+.build.gradle
+----
+version = "0.2"
+----
+
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+Edit `pom.xml` and increment the version to `0.2`:
+
+[source, xml]
+.pom.xml
+----
+<version>0.2</version>
+----
+
+:exclude-for-build:
+
+Depending on the Micronaut version you're using, you might also need to update some properties in your build script to update the Docker configuration.
+
+:exclude-for-build:maven
+
+In your `build.gradle`, change the base image to `gcr.io/distroless/cc-debian10` in the `dockerfileNative` block:
+
+[source, groovy]
+.build.gradle
+----
+dockerfileNative {
+    args("-XX:MaximumHeapSizePercent=80")
+    baseImage('gcr.io/distroless/cc-debian10')
+}
+----
+
+:exclude-for-build:
+
+:exclude-for-build:gradle
+
+In the "configuration" block of the "micronaut-maven-plugin" plugin in your `pom.xml`, change the base image to `gcr.io/distroless/cc-debian10` in a new `baseImageRun` element:
+
```

Comment:
Maybe it's worth explaining why it is necessary to switch to a distroless Docker image and what that does under the hoods: building a "mostly static" native image by adding automatically the flag `-H:+StaticExecutableWithDynamicLibC`